extensions/arc/package.json

```diff
@@ -820,33 +862,53 @@
                     {
                       "title": "%arc.postgres.settings.resource.title%",
                       "fields": [
-                        {
-                          "label": "%arc.postgres.server.group.cores.request.label%",
-                          "description": "%arc.postgres.server.group.cores.request.description%",
-                          "variableName": "AZDATA_NB_VAR_POSTGRES_SERVER_GROUP_CORES_REQUEST",
-                          "type": "number",
-                          "min": 1
-                        },
                         {
                           "label": "%arc.postgres.server.group.cores.limit.label%",
```

Comment:
Just converted the test regex validations in postgress to the new RegexValidation type. However, i have not added integer enforcement and limit >= request for these yet as it would be good if these get whetted out by the PG team perhaps.

Reply:
Ah yes for sure.

Reply:
Ping both Brian and Naia to review.

Reply:
@ranasaria Yes, request must be <= limit.  What is "integer enforcement" though?

Reply:
Integer enforcement is for number fields to ensure that only integer values pass through. It was to be used for SQL MIAA cores but postponed that until Azdata itself starts enforcing that check.

Reply:
Are the plans to have cores in pg stop supporting fractional values?

Reply:
Gotcha.  azdata does validate that workers (AZDATA_NB_VAR_POSTGRES_SERVER_GROUP_WORKERS) is an integer, so ADS could validate that.

Reply:
Ok will add a check of is_integer for AZDATA_NB_VAR_POSTGRES_SERVER_GROUP_WORKERS

@Naia, to the best of my knowledge, at least for Postgres there are no plans to stop supporting fractional values for cores. There was some talk fo adding this restriction for SQL MIAA cores.

Reply:
Added is_integer checks for AZDATA_NB_VAR_POSTGRES_SERVER_GROUP_WORKERS  and AZDATA_NB_VAR_POSTGRES_SERVER_GROUP_PORT. Can you confirm that AZDATA errors if that value is not an integer as well? In addition, perhaps there is a min/max for the port that we can put in.

Reply:
yes

```
azdata arc postgres server create -n pg --port 1.23
Service port must be a positive integer, with a minimum of 1.
```